index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <div class="nav_container">
+                <a href="#" class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </a>
+     
+                <nav class="toggle">
+                    <img src="images/Union.png" alt="hamburger menu">
+                </nav>
+              </div>
+        </header>
+
+        <main>
+            <div class="main_container">
+               <h1 class="box1">
+                   I'm Alex Glad to see you!
+               </h1>
+               <p class="box2">
+                I'm a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
+               </p>
+               <div class="box3">
+                   LET'S CONNECT
+               </div>
+               <ul class="box4">
+                    <li class="socials">
+                       <a href="#">
+                         <img src="images/twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                    <li class="socials">
+                      <a href="#">
+                        <img src="images/Linkedin.png" alt="Linkedin">
+                      </a>
+                    </li>
+                    <li class="socials child">
+                      <a href="#">
+                        <img src="images/m.png" alt="M">
+                      </a>
+                    </li>
+                    <li class="socials">
+                      <a href="#">
+                        <img src="images/github.png" alt="github">
+                      </a>
+                    </li>
+                    <li class="socials">
+                      <a href="#">
+                        <img src="images/peace.png" alt="peace">
+                      </a>
+                    </li>
+               </ul>
```

Comment:
- [OPTIONAL] 

Kindly add your social media profile link to the `href` attribute of the `a` tag. The `#` was to demonstrate that you needed to add a link to your social media according to each icon.

Reply:
Alright Thanks, would do that 😊